app/views/assignments/_assignment.html.slim

```diff
@@ -7,12 +7,13 @@ tr
   - if action_name == 'index'
     td = link_to assignment.volunteer, assignment.volunteer
   td= t("simple_form.options.assignment.state.#{assignment.state}") if assignment.state.present?
-  td= l(assignment.created_at, format: :date_only)
+  td= l(assignment.period_start) if assignment.period_start
+  td= l(assignment.period_end) if assignment.period_end
   td
     - if policy(User).show?
-      = link_to assignment.creator, assignment.creator if assignment.creator.present?
+      = link_to assignment.creator.full_name, profile_path(assignment.creator) if assignment.creator.present?
```

Comment:
why is a check here needed if it exists? If a creator would be existing, he couldn't create even an assignment, or did I get this logic wrong?^^

Reply:
Maybe it is in case the user gets deleted

Reply:
`if assignment.creator.present?` can actually be dropped, as an `Assignment` would never be valid without having one.
It could be soft deleted tho, but that case is caught by:
```ruby
belongs_to :creator, -> { with_deleted }, class_name: 'User'
```
So as long as our db is not corrupt, we can always (even if soft deleted) expect Assignment to have a creator.